tests/roles/ironic_adoption/defaultes/main.yaml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Just noticed a typo: the directory is called "defaultes". Ironic adoption is not tested in CI so if you want to have CI feedback, you can let the current CI jobs finish. We should just fix this before merging.

Reply:
Completely missed it, thanks will fix after CI

Reply:
fixed this and the neutron typo that failed the gate